### PR TITLE
fix:  hide small balance token in wallet #2117

### PR DESCRIPTION
**Pull request by lelenei**
#2117

### CODE REVIEW

File: packages/maskbook/src/extension/options-page/DashboardComponents/WalletAssetsTable.tsx [hunk 1]
```diff
@@ -64,99 +71,123 @@ export interface WalletAssetsTableProps extends withClasses<KeysInferFromUseStyl
...
+                </Box>,
+                <Box
+                    sx={{
+                        display: 'flex',
+                        justifyContent: 'flex-end',
+                    }}>
+                    <Typography className={classes.name} color="textPrimary" component="span">
+                        {new BigNumber(
+                            formatBalance(new BigNumber(x.balance), x.token.decimals ?? 0, x.token.decimals ?? 0),
+                        ).toFixed(stableCoinTokens.some((y) => y.address === x.token.address) ? 2 : 6)}
```
Comment (guanbinrui) [inline packages/maskbook/src/extension/options-page/DashboardComponents/WalletAssetsTable.tsx:119]: Please use `isSameAddress` instead.

File: packages/maskbook/src/web3/hooks/useAssetsStableCoinDetailedDebank.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    const { data = [], error_code } = (await response.json()) as Debank.BalanceListResponse
+    if (error_code === 0) return data
+    return []
+}
+
+export function useAssetsStableCoinDetailedDebank() {
+    const chainId = useChainId()
+    return useAsyncRetry(async () => {
+        const data = await fetcher()
+        return data.map((x) => ({
```
Comment (guanbinrui) [inline packages/maskbook/src/web3/hooks/useAssetsStableCoinDetailedDebank.ts:17]: It's not the type of `EtherToken`.

**Comment by guanbinrui** [comment]
Please rebase the master branch to remove type warnings.

File: packages/maskbook/src/extension/options-page/DashboardComponents/WalletAssetsTable.tsx [hunk 3]
```diff
@@ -64,99 +71,127 @@ export interface WalletAssetsTableProps extends withClasses<KeysInferFromUseStyl
...
+                ))}
+        </TableRow>
+    )
+
+    const LessButton = () => (
+        <div
+            className={classes.lessButton}
+            onClick={() => {
+                setMore(!more)
+                setViewLength(more ? 5 : detailedTokens.length)
```
Comment (guanbinrui) [inline packages/maskbook/src/extension/options-page/DashboardComponents/WalletAssetsTable.tsx:160]: Declare constant variables for those magic numbers.

File: packages/maskbook/src/web3/hooks/useAssetsStableCoinDetailedDebank.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    return []
+}
+
+export function useAssetsStableCoinDetailedDebank() {
+    const chainId = useChainId()
+    return useAsyncRetry(async () => {
+        const data = await fetcher()
+        return data.map((x) =>
+            createERC20Token(chainId, formatChecksumAddress(x.id), x.decimals, x.name, x.symbol),
+        ) as ERC20TokenDetailed[]
```
Comment (guanbinrui) [inline packages/maskbook/src/web3/hooks/useAssetsStableCoinDetailedDebank.ts:21]: The type casting can be reduced.

**Review by guanbinrui — APPROVED** [review]

**Comment by lelenei** [comment]
close #2117

**Pull request merged**